README.md

```diff
@@ -439,6 +439,35 @@ Make sure you've completed the following assignments:
 * (Optional) Go to your user dashboard and create a new repository to practice what you've learned here. Feel free to experiment!
 * [Let us know what worked and what needs improvement in this lesson in the CoMSES Education forum](https://forum.comses.net/c/education/28). Was there anything confusing? What can we do better? If you'd like to submit feedback in private, please send an email to us at `education@comses.net`
 
+## 💪 Contributing to this repo
+
+You can train your GitHub skills by contributing to this repo.
```

Comment:
"This repo" (as in, this actual repo, comses-education/github-starter-course) will be different than a `learn-git-<username>` that gets generated when someone joins the GitHub Classroom via the link 

https://classroom.github.com/a/bLyjPEf6

E.g., https://github.com/comses-education/learn-git-alee

So some additional guidance is needed here to avoid confusing GitHub Classroom learners. Are we asking them to contribute to their `learn-git` repo or to this parent repository?

We'll instead need some prose text like 

```
If you would like to improve this lesson, you can contribute to the classroom repository at https://github.com/comses-education/github-starter-course"

We use the GitHub flow to do that ...

```